develop/application-devs/interact/understand-parachain-data.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Understanding Metadata
+description: Learn how to interact and decipher parachain data (metadata).
+---
+
+Each Polkadot SDK-based network exposes information about its runtime, allowing developers to know granular details regarding the types, pallets, and runtime APIs that a chain may offer.  Metadata files are SCALE-encoded, and allow the development of browser-based, mobile, or desktop applications that can keep up with potential forkless upgrades.  The concept of metadata also aids in ensuring that a single application can support many Polkadot SDK-based chains at once.
+
+## Exposing runtime information as metadata
```

Comment:
```suggestion
## Exposing Runtime Information as Metadata
```
Use Chicago title-style capitalization for titles and headings, which capitalizes the first word plus all other significant words. Same for other headings